src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.serviceimpl.DataConverterImpl;
+import core.basesyntax.service.serviceimpl.FileReaderImpl;
+import core.basesyntax.service.serviceimpl.FileWriterImpl;
+import core.basesyntax.service.serviceimpl.ReportGeneratorImpl;
+import core.basesyntax.service.serviceimpl.ShopServiceImpl;
+import core.basesyntax.stategy.OperationStrategy;
```

Comment:
The package for strategy implementations should be 'strategy', but here it is 'stategy'. There is a typo in the package name.